ast/builtins.go

```diff
@@ -1715,6 +1716,19 @@ var JSONIsValid = &Builtin{
 	Categories: encoding,
 }
 
+var RenderTemplate = &Builtin{
+	Name:        "render_template",
+	Description: "Renders a templated string with given template variables injected. See https://pkg.go.dev/text/template for templating examples.",
```

Comment:
Nit: If you can explain a bit what the built-in does instead of the including a link that would be better.